lago_images/images.py

```diff
@@ -119,6 +119,36 @@ def custom_build_action(self, *args, **kwargs):
             commands_file=self.spec.commands_file
         )
 
+        if self.spec.size != 0:
+            build_utils.resize_disk_image(
+                dst_image=self.dst_path,
+                new_size=self.spec.size,
+            )
+
+            build_utils.copy_image_with_cp(
```

Comment:
Why is this needed? the previous step already creates a copy of the original disk.